spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -0,0 +1,281 @@
+describe('GOVUK.OptionSelect', function() {
+
+  var $optionSelectHTML, optionSelect;
+
+  beforeEach(function(){
+
+    optionSelectFixture = '<div class="govuk-option-select" tabindex="0">'+
```

Comment:
Initially I had the fixtures in a separate file being loaded in, but currently chrome blocks file:// URIs from reading other file:// URIs, which means they can't be loaded in chrome unless you switch --allow-file-access-from-files on. (more on this here: https://github.com/velesin/jasmine-jquery#cross-domain-policy-problems-under-chrome) Introducing this felt like a major gotcha for developers using chrome coming to the project so I put the html in the test to avoid future confusion.